.github/workflows/eval-aliases.yml

```diff
@@ -0,0 +1,33 @@
+name: Eval aliases
+
+on:
+  pull_request_target:
+
+permissions: {}
+
+jobs:
+  get-merge-commit:
+    uses: ./.github/workflows/get-merge-commit.yml
```

Comment:
Longterm I am wondering instead of having a get-merge-commit for each workflow if we could not have a re-usable step instead so we can save one VM instance each time we need this.

Reply:
I think it might be worth it because we have 4x other workflows that do the same thing.

Reply:
Ok. Looks like github doesn't support this than.